Makefile

```diff
@@ -39,15 +39,34 @@ DEFAULT_IMAGE_REGISTRY := quay.io/stackrox-io
 BUILD_IMAGE_VERSION=$(shell sed 's/\s*\#.*//' BUILD_IMAGE_VERSION)
 BUILD_IMAGE := $(DEFAULT_IMAGE_REGISTRY)/apollo-ci:$(BUILD_IMAGE_VERSION)
 
+GOINSTALL := GOARCH="" go install
+DOCKERBUILD := $(CURDIR)/scripts/docker-build.sh
+
+DB_DOCKERBUILD_ARGS =
+ifeq ($(GOARCH),s390x)
+        DB_DOCKERBUILD_ARGS = \
+                --build-arg="RPMS_REGISTRY=quay.io" \
+                --build-arg="RPMS_BASE_IMAGE=centos/centos" \
+                --build-arg="RPMS_BASE_TAG=stream9" \
+                --build-arg="BASE_IMAGE=ubi9-minimal"
+endif
+
+# Update the arch to arm64 but only for Macs running on Apple Silicon (M1)
```

Comment:
This comment doesn't make sense now that we support multiple architectures (this is from https://github.com/stackrox/stackrox/commit/d1af797460eb38a2b9b7eee4cd34a7a9bcc9301b which was a long time ago)